tests/lodash-deep-extras-tests.js

```diff
@@ -289,232 +228,165 @@ describe(
 
```

Comment:
I was referring to the 'isInteger' test it didn't get the updated spacing applied